cdrs-tokio/src/cluster/session.rs

```diff
@@ -978,6 +985,7 @@ pub enum SessionBuildError {
 /// Builder for easy `Session` creation. Requires static `LoadBalancingStrategy`, but otherwise, other
 /// configuration parameters can be dynamically set. Use concrete implementers to create specific
 /// sessions.
+#[async_trait]
```

Comment:
We should not use async_trait. The return types mess up the docs, add memory allocation overhead and can cause problems with storing.

Reply:
Either create an explicit future type or use a BoxFuture. async_trait is not that great in library code.